include/seqan3/io/sequence/sequence_file_in_options.hpp

```diff
@@ -51,9 +51,6 @@ namespace seqan3
 template <typename sequence_legal_alphabet_, bool seq_qual_combined>
 struct sequence_file_in_options
 {
-    //!\brief Export the (possibly larger) legal alphabet to the format.
-    using sequence_legal_alphabet = sequence_legal_alphabet_;
-
```

Comment:
If you remove this, please rename the template-parameter so that the trailing `_` is also removed.